cpp/examples/symreg/symreg_example.cpp

```diff
@@ -337,8 +338,8 @@ int main(int argc, char* argv[])
 
   /* ======================= Reset data ======================= */
 
-  raft::deallocate(d_finalprogs, stream);
+  curr_mr->deallocate(d_finalprogs, params.population_size, stream);
```

Comment:
Not completely in love with this one because it bypasses the safety of the container / buffer

Reply:
Thanks for the review! From memory, I believe I couldn't use the `rmm::device_uvector` because the type isn't trivially copyable.